tests/data/auxtel/test_sequence1.yaml

```diff
@@ -0,0 +1,17 @@
+"- em_exp_time: 5\n  em_n_exp: 1\n  entrance_slit_width: 4.5\n  exit_slit_width: 4.5\n\
+  \  exp_time: 3\n  fs_exp_time: 5\n  fs_n_exp: 1\n  grating: 1\n  n_exp: 5\n  spec_res:\
+  \ -1\n  wavelength: 550.0\n- em_exp_time: 5\n  em_n_exp: 1\n  entrance_slit_width:\
+  \ 4.5\n  exit_slit_width: 4.5\n  exp_time: 3\n  fs_exp_time: 5\n  fs_n_exp: 1\n\
+  \  grating: 1\n  n_exp: 5\n  spec_res: -1\n  wavelength: 570.0\n- em_exp_time: 5\n\
+  \  em_n_exp: 1\n  entrance_slit_width: 4.5\n  exit_slit_width: 4.5\n  exp_time:\
+  \ 3\n  fs_exp_time: 5\n  fs_n_exp: 1\n  grating: 1\n  n_exp: 5\n  spec_res: -1\n\
+  \  wavelength: 590.0\n- em_exp_time: 5\n  em_n_exp: 1\n  entrance_slit_width: 4.5\n\
+  \  exit_slit_width: 4.5\n  exp_time: 3\n  fs_exp_time: 5\n  fs_n_exp: 1\n  grating:\
+  \ 1\n  n_exp: 5\n  spec_res: -1\n  wavelength: 610.0\n- em_exp_time: 5\n  em_n_exp:\
+  \ 1\n  entrance_slit_width: 4.5\n  exit_slit_width: 4.5\n  exp_time: 3\n  fs_exp_time:\
+  \ 5\n  fs_n_exp: 1\n  grating: 1\n  n_exp: 5\n  spec_res: -1\n  wavelength: 630.0\n\
+  - em_exp_time: 5\n  em_n_exp: 1\n  entrance_slit_width: 4.5\n  exit_slit_width:\
+  \ 4.5\n  exp_time: 3\n  fs_exp_time: 5\n  fs_n_exp: 1\n  grating: 1\n  n_exp: 5\n\
+  \  spec_res: -1\n  wavelength: 650.0\n- em_exp_time: 5\n  em_n_exp: 1\n  entrance_slit_width:\
+  \ 4.5\n  exit_slit_width: 4.5\n  exp_time: 3\n  fs_exp_time: 5\n  fs_n_exp: 1\n\
+  \  grating: 1\n  n_exp: 5\n  spec_res: -1\n  wavelength: 670.0\n"
```

Comment:
That's odd. How were you loading and printing it?

Reply:
Never mind. I was doing it in the wrong way, instead of using this:

```
with open('test_sequence1.yaml', "r") as file:
    sequence = yaml.safe_load(file)

```
I was using a workaround (damm it) on the jupyter notebook copying the content of the yaml file into the notebook.